static/oas/v1/openapi.yml

```diff
@@ -10,23 +10,24 @@ info:
 paths:
   /auth:
     $ref: "./auth/auth.yml#/paths/auth"
-  /shotsongoal:
-    $ref: "./stats/shotsongoal.yml#/paths/shotsOnGoal"
-  /penaltyTypes:
-    $ref: "./stats/penalties.yml#/paths/penaltyTypes"
+  /goals:
+    $ref: "./stats/goal.yml#/paths/goals"
   /leagues:
     $ref: "./leagues/leagues.yaml#/paths/leagues"
+  /logos:
+    $ref: "./groups/logos.yml#/paths/~1logos"
+  /logos/{id}:
+    $ref: "./groups/logos.yml#/paths/~1logos~1%7Bid%7D"
   /penalties:
     $ref: "./stats/penalties.yml#/paths/penalties"
-  /user:
-    $ref: "./users/users.yml#/paths/user"
-
-  # /[URL path]
-  #$ref: "./[Relative path starting from v1]#/paths/[yml path]"
-  /goals:
-    $ref: "./stats/goal.yml#/paths/goals"
+  /penaltyTypes:
+    $ref: "./stats/penalties.yml#/paths/penaltyTypes"
   /seasons:
     $ref: "./season/season.yml#/paths/seasons"
```

Comment:
You will need to change this line to `./groups/season.yml...`

Reply:
No the YAML file for season is still under its own folder currently
`static/oas/v1/season/season.yml`